### PR TITLE
addtional string on

### PR DESCRIPTION
**Pull request by TokyoYoshida**
this is test pull request.

### CODE REVIEW

File: sinatora_hello_world.rb [hunk 1]
```diff
@@ -2,5 +2,5 @@
 require 'sinatra'
 
 get '/' do
-  'Hello world!'
+  'Hello world! this is test.'
```
Comment (jugemjugemjugem) [inline sinatora_hello_world.rb:5]: ok good.

Comment (jugemjugemjugem) [inline sinatora_hello_world.rb:5]: this is test conversation

**Review by jugemjugemjugem — COMMENTED** [review]
this is test summary

**Comment by TokyoYoshida** [comment]
this is test comment.

**Pull request merged**